Modules/Filtering/Smoothing/include/itkRecursiveGaussianImageFilter.hxx

```diff
@@ -342,7 +342,29 @@ RecursiveGaussianImageFilter<TInputImage, TOutputImage>::PrintSelf(std::ostream
   Superclass::PrintSelf(os, indent);
 
   os << "Sigma: " << m_Sigma << std::endl;
-  os << "Order: " << m_Order << std::endl;
+  switch (m_Order)
```

Comment:
@mseng10  Mathew you seemed to have missed one case of providing the stream operators for the GaussianOrderEnum.   Please add the correct operator<<  for this type ASAP so that we can avoid  specialized code like is included here.

Reply:
@hjmjohnson This enum class already has an existing operator function, but I think it isn't being properly called. Should I create a separate PR for my changes?